.drone/scripts/create_tag.sh

```diff
@@ -19,6 +19,13 @@ chmod 500 "/${HOME}/.ssh/"* -R
 # Get current package version
 version="$(cat .data.json | jq -r '.current_pkgver + "-" + .current_pkgrel')"
 
+# Update debian version
+DEBVERSION="$(cat debian/changelog | cut -f2 -d" " - | grep "(" | cut -f2 -d"(" | cut -f1 -d")")"
+
+if [ $vresion != $DEBVERSION ]; then
+  dch -v $VERSION -i "Initial release (Closes: #998039)."
+fi
+
```

Comment:
Is this supposed to be pushing the modified changelog back to this repo?

Reply:
Yes.